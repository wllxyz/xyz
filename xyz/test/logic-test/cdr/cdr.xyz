<i>-->""==><o>-->($CDR, ($CAR, ($IGNORE, ( ($ADD, "0", "1") ) ) ) );
