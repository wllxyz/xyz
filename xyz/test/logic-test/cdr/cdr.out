$LEFT_QUOTE0$SEPERATOR1$RIGHT_QUOTE